src/upgrader.ts

```diff
@@ -318,7 +330,7 @@ export class DefaultUpgrader extends EventEmitter<UpgraderEvents> implements Upg
       maConn,
       upgradedConn,
       muxerFactory,
-      remotePeer
+      remotePeer,
```

Comment:
I think this will upset the linter.  You can run `npm run lint` locally to check.

```suggestion
      remotePeer
```